motion_planner_state planner

---
uint32 error_code
